src/EntityFramework7.Npgsql/project.json

```diff
@@ -14,8 +14,8 @@
     "keyFile": "../../Npgsql.snk"
   },
   "dependencies" : {
-    "EntityFramework.Core": "7.0.0-rc1-16105",
-    "EntityFramework.Relational": "7.0.0-rc1-16105",
+    "EntityFramework.Core": "7.0.0-*",
```

Comment:
I'm not sure this is a very good idea - EF7 moves forward a lot and there's considerable API breakage. Up to now, I've always built against a specific EF7 version, and I think it's a good idea to lock against that version - this way we know that EF7.Npgsql builds. Any specific trouble this is causing you?